### DISCUSSION TITLE
Contributing: manual

### DISCUSSION THREAD

**Discussion by dmyersturnbull** [Keriew/augustus#581] [General]
I noticed the manual for 3.1 only covers the new features. I'd be happy to improve the manual if that would be helpful.

**Q: Where is the source for the manual that I should work off of?**

### Addendum -- to-do items I'm considering
- [ ] Merge relevant sections of 2.0 and 3.1 manuals into the current working manual (4.0?).
- [ ] Add content covering C3 basics, like in the original C3 manual.
- [ ] Improve typesetting (e.g. spacing, fonts, links in the table of contents).
- [ ] Add a link into the C3 menu it can be found easily.
- [ ] Maybe eventually: in-game search or other nav in-game.

**Comment by crudelios** [comment]
This would be very helpful, it's actually one of the pending things that no one had the time to do yet.

Do you have Discord?

**Reply by dmyersturnbull** [reply]
Yep, I'll send a message on the augustus-development channel.
